source/lexbor/html/tree/error.h

```diff
@@ -97,6 +97,8 @@ lxb_html_tree_error_id_t;
 
 typedef struct {
     lxb_html_tree_error_id_t id;
+    const lxb_char_t *token_begin;
```

Comment:
Hi @nielsdos 

Style (name of members on one line):
```C
typedef struct {
    lxb_html_tree_error_id_t id;
    const lxb_char_t         *token_begin;
    ...
```

Rename: `token_begin` -> `begin` and `token_end` -> `end`.

Reply:
Thanks for reviewing, I did the requested changes.